infer_segmentation.py

```diff
@@ -166,18 +208,33 @@ def main(gcp_bucket, stack_id, model_id, prediction_threshold):
         '--gcp-bucket',
         type=str,
         help='The GCP bucket where the raw data is located and to use to store the processed stacks.')
+    argparser.add_argument(
+        '--model-id',
+        type=str,
+        help='The model ID.')
+    argparser.add_argument(
+        '--background-class',
+        type=int,
+        default=None,
+        help='For this model, indicate background class if used during model training, to exclude background overlay.')
     argparser.add_argument(
         '--stack-id',
         type=str,
         help='The stack ID (must already be processed).')
     argparser.add_argument(
-        '--model-id',
+        '--image-ids',
         type=str,
-        help='The model ID.')
+        default=None,
+        help='For these images, the corresponding stack ID (must already be processed).')
     argparser.add_argument(
         '--prediction-threshold',
         type=float,
         default=0.5,
         help='Threshold to apply to the prediction to classify a pixel as part of a class.')
+    argparser.add_argument(
```

Comment:
was this the value we were going to make binary rather than none or string?

Reply:
yup just didnt get to it yet. on it now